### PR TITLE
Create workshops page

### PR DESCRIPTION
**Pull request by nathangong**
Addresses #38

### CODE REVIEW

File: src/components/workshops/WorkshopData.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    title: "Intro to USACO Webinar",
+    description: "Experienced USACO contestants explain everything you need to know about getting started with USACO.",
+    date: dayjs('Nov 28 2020 5:00:00 PM PDT'),
+    url: "intro-to-usaco"
+  },
+]
+
+workshops.sort((a, b) => -a.date.diff(b.date));
+const now = dayjs();
+console.log(workshops);
```
Comment (maggie-j-liu) [inline src/components/workshops/WorkshopData.ts:41]:
```suggestion
```

File: src/components/workshops/WorkshopList.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            <Image
+              objectPosition="center center"
+              src={latestWorkshop.photo}
+              alt={latestWorkshop.title}
+              placeholder="blur"
+              objectFit="contain"
+            />
+          </div>
+          <div className="flex justify-center mt-8">
+            <a
```
Comment (maggie-j-liu) [inline src/components/workshops/WorkshopList.tsx:34]: wrap the `Link` from [`next/link`](https://nextjs.org/docs/api-reference/next/link) around the `a` tag here

File: src/components/workshops/WorkshopList.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            <div className="mb-10 space-y-5 sm:space-y-4">
+              <h2 className="flex justify-center font-extrabold tracking-tight text-4xl mb-8">
+                {section.title}
+              </h2>
+              <ul
+                role="list"
+                className="mb-16 mx-auto space-y-8 sm:grid md:grid-cols-2 sm:gap-8 sm:space-y-0 lg:grid-cols-2 xl:grid-cols-2"
+              >
+                {section.workshops.map(workshop => (
+                  <Link
```
Comment (maggie-j-liu) [inline src/components/workshops/WorkshopList.tsx:54]: wrap the link around an `a` tag

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

File: src/components/workshops/WorkshopList.tsx (hunk 2 above)
Comment (nathangong) [reply src/components/workshops/WorkshopList.tsx:34]: Would there be any need for that since this opens up a new tab?

Comment (nathangong) [reply src/components/workshops/WorkshopList.tsx:34]: Nvm I can prob make this open up in the same tab too for consistency

File: package.json [hunk 4]
```diff
@@ -19,7 +19,8 @@
     "axios": "^0.20.0",
     "classnames": "^2.2.6",
     "cssnano": "^4.1.10",
-    "firebase": "8.10.0",
+    "dayjs": "^1.10.7",
+    "firebase": "^9.1.1",
```
Comment (maggie-j-liu) [inline package.json:23]: downgrade firebase back to 8.10.0, that's probably why the build broke

**Review by maggie-j-liu — APPROVED** [review]

**Comment by nathangong** [comment]
Oh wait mobile looks a bit weird, lemme fix that rn

**Pull request merged**